bowtie/_cli.py

```diff
@@ -571,6 +571,53 @@ def do_not_validate(*ignored: SchemaResource) -> Callable[..., None]:
     return lambda *args, **kwargs: None
 
 
+def make_remote_data_getter():
+
+    def get_remote_data(source_url: str) -> list[tuple[str, str | int]]:
```

Comment:
The extra level of callable here should be unnecessary -- you don't need both `make_remote_data_getter` and `get_remote_data`, just `get_remote_data` is fine (same for `no_remote_data` and the `lambda`, you just need one level of callable.